terraform/modules/upload-service/sqs.tf

```diff
@@ -0,0 +1,64 @@
+resource "aws_sqs_queue" "upload_queue" {
+  name                      = "pre_checksum_upload_queue"
+  delay_seconds             = 90
+//  Queue visibility timeout must be larger than (triggered lambda) function timeout
+  visibility_timeout_seconds = 360
+  max_message_size          = 2048
```

Comment:
Please include a comment about what max_message_size means and why this value was chosen